### ISSUE TITLE
Multiple Imputation - Continuous Data MNAR	- Delta Adjustment/Tipping Point

### ISSUE THREAD

**Issue by statasaurus** [PSIAIMS/CAMIS#46]

**Comment by SuperPennyP** [comment]
@andreaczhang I'm interested in this issue, can I take the SAS code part?

**Comment by andreaczhang** [comment]
> @andreaczhang I'm interested in this issue, can I take the SAS code part?

Of course! Sorry for the late reply, I was on holiday. Go ahead, thank you! @SuperPennyP

**Comment by DrLynTaylor** [comment]
Yannick's intern may be able to work on this summer 2025
